content/docs/replication/high-availability/powerstore-metro.md

```diff
@@ -0,0 +1,60 @@
+---
+title: PowerStore Metro
+linktitle: PowerStore Metro
+weight: 2
+description: >
+  High Availability support for CSI PowerStore
+---
+
+## PowerStore Metro Architecture
+
+![metro architecture diagram](../../powerstore-metro.png)
+
+In PowerStore Metro configurations:
+* The application host can write data to both sides of the Metro volume. 
+* The devices in the Metro volume are configured with the same external device identity, including the geometry and device WWN.
+* When Metro is configured on the volume, the PowerStore system from which the metro source is configured is automatically set as preferred and the other is configured as non-preferred.
+
+With respect to Kubernetes, the PowerStore Metro mode works in single cluster scenarios. In the Metro, both the arrays—[arrays with metro link setup between them](../../../deployment/helm/modules/installation/replication/powerstore/#on-storage-array)—involved in the replication are managed by the same `csi-powerstore` driver. The replication is triggered by creating a volume using a `StorageClass` with metro-related parameters.
+The driver on receiving the metro-related parameters in the `CreateVolume` call creates a metro replicated volume and the details about both the volumes are returned in the volume context to the Kubernetes cluster. So, the Persistent Volume (PV) created in the process represents a pair of metro replicated volumes. When a `PV`, representing a pair of metro replicated volumes, is claimed by a pod, the host treats each of the volumes represented by the single `PV` as a separate data path. The switching between the paths, to read and write the data, is managed by the multipath driver. The switching happens automatically, as configured by the user—in round-robin fashion or otherwise—or it can happen if one of the paths goes down. For details on Linux multipath driver setup, [click here](../../../deployment/helm/drivers/installation/powerstore#linux-multipathing-requirements).
```

Comment:
```suggestion
The driver on receiving the metro-related parameters in the `CreateVolume` call creates a metro replicated volume and the details about both the volumes are returned in the volume context to the Kubernetes cluster. The Persistent Volume (PV) created in the process represents a pair of metro replicated volumes. When a `PV`, representing a pair of metro replicated volumes, is claimed by a pod, the host treats each of the volumes represented by the single `PV` as a separate data path. The switching between the paths, to read and write the data, is managed by the multipath driver. The switching happens automatically, as configured by the user—in round-robin fashion or otherwise—or when one of the paths goes down. For details on Linux multipath driver setup, [click here](../../../deployment/helm/drivers/installation/powerstore#linux-multipathing-requirements).
```